src/GovernorCountingFractional.sol

```diff
@@ -151,7 +168,23 @@ abstract contract GovernorCountingFractional is Governor {
     /**
      * @dev Count votes with fractional weight.
      *
-     * We expect `voteData` to be three packed uint128s, i.e. encodePacked(againstVotes, forVotes, abstainVotes)
+     * `voteData` is expected to be three packed uint128s, i.e.
+     * encodePacked(againstVotes, forVotes, abstainVotes).
+     *
+     * This function can be called multiple times for the same account and
+     * proposal, i.e. partial/rolling votes are allowed. For example, an account
+     * with total weight of 10 could call this function three times with the
+     * following vote data:
+     *   - against: 1, for: 0, abstain: 2
+     *   - against: 3, for: 1, abstain: 0
+     *   - against: 1, for: 1, abstain: 1
+     * The result of these three calls would be that the account casts 5 votes
+     * AGAINST, 2 votes FOR, and 3 votes ABSTAIN on the proposal. Though
+     * partial, votes are still final once cast and cannot be changed or
+     * overidden. Subsequent partial votes simply add to existing totals.
+     *
+     * Note that if partial votes are cast, all remaining weight must be cast
```

Comment:
Nit: Might rephrase to `Note that **once** partial votes are cast,`

Reply:
I tried this but honestly preferred the existing phrasing. "Once" sounds a little presumptive to my ear, like we're assuming partial votes will be cast